templates/wsgi.py.epp

```diff
@@ -1,11 +1,12 @@
+<%- | Stdlib::Absolutepath $base_dir,
+| -%>
+# THIS FILE IS MANAGED BY PUPPET
 from __future__ import absolute_import
 
 import os
 import sys
 
-os.environ['PUPPETBOARD_SETTINGS'] = '<%= @basedir %>/puppetboard/settings.py'
-activate_this = '<%= @basedir %>/virtenv-puppetboard/bin/activate_this.py'
-exec(compile(open(activate_this).read(), activate_this, 'exec'), dict(__file__=activate_this))
+os.environ['PUPPETBOARD_SETTINGS'] = '<%= $base_dir %>/puppetboard/settings.py'
 
 me = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Perhaps it can be a static file rather than a template:

```suggestion
me = os.path.dirname(os.path.abspath(__file__))

os.environ['PUPPETBOARD_SETTINGS'] = os.path.join(me, 'settings.py')
```

Untested, but if I read the code correct, this is equivalent

Reply:
tested this locally and applied the patch, thanks!